loops.js

```diff
@@ -18,6 +21,23 @@
 // klaar!
  // ==========================================
 
+// optie 1
+for (let i = 0; i <5; i++) {
+    if (i=== 4) {
+        console.log("Klaar!")
+    } else {
+        console.log("loop")
+    }
+}
+
+//optie 2
+for (let i = 0; i <4; i++){
+    console.log("loop")
+    if (i === 3)
+    console.log("Klaar!")
+}
+
```

Comment:
Voor alle strings in javascript gebruik je single quotes. 
Verder echt super dat je twee werkende opties hebt gegeven.